contrib/pyzfs/libzfs_core/bindings/libzfs_core.py

```diff
@@ -101,18 +102,19 @@
     int lzc_load_key(const char *, boolean_t, uint8_t *, uint_t);
     int lzc_promote(const char *, nvlist_t *, nvlist_t **);
     int lzc_receive(const char *, nvlist_t *, const char *, boolean_t,
-        boolean_t, int);
+        boolean_t, boolean_t, int);
     int lzc_receive_one(const char *, nvlist_t *, const char *, boolean_t,
-        boolean_t, boolean_t, int, const dmu_replay_record_t *, int,
+        boolean_t, boolean_t, boolean_t, int, const dmu_replay_record_t *, int,
         uint64_t *, uint64_t *, uint64_t *, nvlist_t **);
     int lzc_receive_resumable(const char *, nvlist_t *, const char *,
-        boolean_t, boolean_t, int);
+        boolean_t, boolean_t, boolean_t, int);
     int lzc_receive_with_cmdprops(const char *, nvlist_t *, nvlist_t *,
-        const char *, boolean_t, boolean_t, boolean_t, int,
+        const char *, boolean_t, boolean_t, boolean_t, boolean_t, int,
         const dmu_replay_record_t *, int, uint64_t *, uint64_t *, uint64_t *,
         nvlist_t **);
     int lzc_receive_with_header(const char *, nvlist_t *, const char *,
-        boolean_t, boolean_t, boolean_t, int, const dmu_replay_record_t *);
+        boolean_t, boolean_t, boolean_t, boolean_t, int,
+        const dmu_replay_record_t *);
     int lzc_release(nvlist_t *, nvlist_t **);
```

Comment:
Are we allowed to change this interface or are these functions considered "stable" (with maybe the exception of `lzc_receive_with_cmdprops`) across platforms?